llarp/config/config.cpp

```diff
@@ -1250,6 +1250,15 @@ namespace llarp
         [this](std::string arg) { rpc_addr = oxenmq::address(arg); });
 
     // Deprecated options:
+    conf.define_option<std::string>("lokid", "jsonrpc", RelayOnly, Deprecated, [](std::string arg) {
```

Comment:
This should not be marked `Deprecated`: that tag means the option is silently ignored, but this option needs to be a fatal error if specified and not ignored.  It should be `Hidden`, however, so that it doesn't go into generate configs.